notebooks/mnist_data_analysis.ipynb

```diff
@@ -0,0 +1,209 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b8280ff3",
+   "metadata": {},
+   "source": [
+    "# MNIST data analysis"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5ba02a8",
+   "metadata": {},
+   "source": [
+    "### Loading MNIST dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "8ebe0648",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from torchvision import datasets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "2eec81ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DOWNLOAD = False  # whether to download dataset; if not downloaded, use DOWNLOAD = True"
```

Comment:
You could set it to True by default in the following cell. If the datset is present it will not be re-downloaded even if `download==True`

Reply:
Done